spec/models/user_spec.rb

```diff
@@ -1,23 +1,16 @@
-require 'spec_helper'
```

Comment:
This is the only new change in this file:
```ruby
describe 'indices' do
  it { is_expected.to have_db_index(:persistence_token).unique(true) }
end
 ```
 Refer to https://github.com/GSA/search-gov/pull/825/commits/338edfb6dc14424a00026d7c565b9c9a535216f2 for changes made only to placate Rubocop. (Note that this does not resolve every offense in that file - I timeboxed the cleanup to fix some low-hanging fruit, which as you can see was still an extensive set of changes...)